timemachine/cpp/src/neighborlist.cu

```diff
@@ -116,8 +116,8 @@ std::vector<std::vector<int> > Neighborlist<RealType>::get_nblist_host(
     );
 
     cudaDeviceSynchronize();
-
     const int B = this->B(); //(N+32-1)/32;
+    // Q:: What does Y stand for? Blocks divided by 32?
     const int Y = this->Y(); //(B+32-1)/32;
```

Comment:
Don't remember, but I'm not sure if Y is even used in this function. (It's used in the constructor)